.buildkite/pipelines/metrics_web_pipeline.yml

```diff
@@ -4,21 +4,39 @@ steps:
     commands:
       - flutter channel beta
       - flutter config --enable-web
+    plugins:
+      - docker-compose:
+          config: ".buildkite/compose-files/flutter-docker-compose.yml"
+          run: flutter
+          workdir: /monorepo
 
   - label: "Flutter analyze"
     key: "analyze"
     depends_on: "configure"
     commands:
       - cd metrics/web
       - flutter analyze
+    plugins:
+      - docker-compose:
+          config: ".buildkite/compose-files/flutter-docker-compose.yml"
+          run: flutter
+          workdir: /monorepo
 
   - label: "Flutter test"
     key: "tests"
     depends_on: "analyze"
+    allow_dependency_failure: true
```

Comment:
Should we make all pipelines run tests even if analyze step fails? The reason is to make all pipelines similar - if we want to run tests regardless of the `analyze` step result, then we should probably do this for all pipelines.